docs/design/3_2_multirotor_v2.tex

```diff
@@ -1,26 +1,231 @@
+
+The 350 mm frames are also popular amongst consumer products such as the DJI Phantom series\cite{dji-phantom-3-specs} as seen in Figure~\ref{fig:djiphantom}. A frame size of 350 mm would constrain our ability to mount larger hardware --- posing a significant risk to integrating computing platform hardware. Additionally, the 350 mm size also limits the propeller maximum size. The largest quadcopter configurations have an MMDS of 1000 mm to allow for extremely large payload capacities such as the one shown in Figure~\ref{fig:djis1000}. However, these are more commonly used for industrial or military applications.
+
+\paragraph{Customized Add-ons}
+% TODO: add CAD parts
+
 \subsubsection{Propulsion}
+
+The propulsion system covers the majority of the multirotor parts list and determines the flight performance of the multirotor. We refer to this subsystem as the ``Drive".
+
 \paragraph{Motor Type}
+
+\underline{\textit{Brushed or brushless?}}
+Multirotors are electrically powered by a battery --- a DC source. Two common types of DC motors are  brushed motors and brushless DC (BLDC) motors. Brushed motors commutate mechanically using brushes whereas brushless motors commutate electrically using electronic controllers that switches phases in the stator coils.
+
+We use  BLDC motors for their extended usable lifespan and efficiency. Brushed motors are not appropriate for our applications for its excessive noise, friction, spark, and heat.
+
+\underline{\textit{Inrunner or outrunners?}}
+
+BLDC motors further consists of two variants: \textit{inrunner} and \textit{outrunner}. 
+
+We choose outrunner BLDC motors for its higher torque output at lower speed  (i.e. outrunner motors have low KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). The lower speed of the rotor is beneficial for two reasons. One, a motor running at a lower speed is more efficient and produces less high frequency vibrations. Two, the attached propeller can be much larger and cut through the air at a lower speed, since drag forces acting on the blades are proportional to the velocity squared. Outrunner motors do require spin-up time as the rotor is more massive and has more inertia, but this compromise is acceptable as the main operating mode of the multirotor is hovering, which requires little agility. 
```

Comment:
```suggestion
We choose outrunner BLDC motors for their higher torque output at lower speed  (i.e. outrunner motors have low KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). The lower speed of the rotor is beneficial for two reasons. One, a motor running at a lower speed is more efficient and produces fewer high frequency vibrations. Two, the attached propeller can be much larger and cut through the air at a lower speed, since drag forces acting on the blades are proportional to the velocity squared. While outrunner motors do require additional spin-up time as the rotor is more massive and has more inertia, this compromise is acceptable as the main operating mode of the multirotor is hovering, which requires little agility. 
```